.github/workflows/scripts/check_translation_usage.py

```
@@ -0,0 +1,424 @@
+            flattened translation keys.
+
+    Raises:
+        FileNotFoundError: If the locales directory is missing, no locale files
+            are found, or a translation.json file contains invalid JSON and
+            allow_missing is False.
+    """
+    base = Path(locales_dir)
+    if not base.exists() or not base.is_dir():
+        raise FileNotFoundError(
+            f"Locales directory not found: {locales_dir!s}"
+        )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)